src/nav/style-guide.yml

```diff
@@ -1,119 +1,128 @@
-      - title: Edit checklist
-        path: /docs/style-guide/processes-procedures/docs-site-edit-checklist
-      - title: Rename or redirect docs
-        path: /docs/style-guide/processes-procedures/rename-or-redirect-document
-      - title: Delete a document
-        path: /docs/style-guide/processes-procedures/delete-document
-      - title: Embed images
-        path: /docs/style-guide/processes-procedures/embed-images
-      - title: Embed videos
-        path: /docs/style-guide/processes-procedures/embed-videos
-      - title: Update the home page
-        path: /docs/style-guide/processes-procedures/edit-homepage
-  - title: Tech writer processes
-    path: /docs/style-guide/writer-workflow
-    pages:
-      - title: Get around GitHub
-        path: /docs/style-guide/writer-workflow/github-intro
-      - title: Tech writer workflow
-        path: /docs/style-guide/writer-workflow/tech-writer-workflow
-      - title: Peer editor workflow
-        path: /docs/style-guide/writer-workflow/peer-editor-workflow
-      - title: GitHub troubleshooting
-        path: /docs/style-guide/writer-workflow/github-troubleshooting
-  - title: Article templates
-    path: /docs/style-guide/article-templates
-    pages:
-      - title: Basic doc template
-        path: /docs/style-guide/article-templates/basic-doc-template
-      - title: Agent release notes template
-        path: /docs/style-guide/article-templates/agent-release-notes-template-123
-      - title: What's new? guidelines
-        path: /docs/style-guide/article-templates/whats-new-style-guidelines
-      - title: Troubleshooting docs guide
-        path: /docs/style-guide/article-templates/troubleshooting-docs-guide
-      - title: Data dictionary style guidelines
-        path: /docs/style-guide/article-templates/data-dictionary-style-guidelines
-      - title: API tutorial template
-        path: /docs/style-guide/article-templates/api-tutorial-template
-      - title: Agent API guide template
-        path: /docs/style-guide/article-templates/agent-api-guide-template
-      - title: apiStyleGuidelines (Example agent API)
-        path: /docs/style-guide/article-templates/apistyleguidelines-example-agent-api
-      - title: GraphQL API tutorial template
-        path: /docs/style-guide/article-templates/graphql-api-tutorial-template
-      - title: Landing page template
-        path: /docs/style-guide/article-templates/landing-page-template
-      - title: Agent release notes template 1.2.3
-        path: /docs/style-guide/article-templates/agent-release-notes-template-123
+    - title: Writing great content at New Relic
+      path: /docs/style-guide/writing-strategies
+      pages: 
+        - title: Introduction to the style guide
+          path: /docs/style-guide/writing-strategies/introduction-style-guide
+        - title: Voice, tone, and style
+          path: /docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic
```

Comment:
I would love to see the 5Qs doc in the "Writing great content" section but defer to you,

Reply:
I have that thought as well. I was hoping to move it up after a review for docs only info, but there are other places where I haven't been able to do that yet, so why wait on this one! Thanks.